### DISCUSSION TITLE
[Feature Request] Support Conversations APIs (HubSpot) natively with incremental mode (append+dedup)

### DISCUSSION THREAD

**Discussion by carlos-yolo** [airbytehq/airbyte#45124] [Connector Ideas and Features]
Hello!

As the title says, I was wondering if this could be done. It would help reduce ingestion costs and post-processing from the current custom connector we're using in full overwrite mode so those resources could be used elsewhere (e.g. another data source). 

Please let me know if I may somehow help.

Have a great day,

Carlos

**Comment by Arnaud-MARIE** [comment]
@carlos-yolo did you solve it with a custom connector?
I'm having a lot a issue to build my own.
